pipelines/productionBuild.yml

```diff
@@ -64,14 +64,6 @@ extends:
             script: |
               echo "##vso[task.setvariable variable=JAVA_HOME]$(JAVA_HOME_11_X64)"
               echo "##vso[task.setvariable variable=PATH]$(JAVA_HOME_11_X64)\bin;$(PATH)"
-        - task: PowerShell@2
```

Comment:
I don't think this should be removed as it acts as validation in the pipeline to ensure the version has been bumped.